packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -32,10 +32,11 @@ module.exports = {
       const isEmail = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(params.identifier);
 
       // Set the identifier to the appropriate query field.
+      const identifier = params.identifier.toLowerCase();
```

Comment:
Why do you need to lowercase the identifier at this line? It should only concern the email. Also, if we do that we have to be sure that the stored email is lowercased during the registration process or that our query whatever the connector (strapi-bookshelf, strapi-mongoose) are making a query that is case-insensitive.

Reply:
The identifier is lowercased here:
https://github.com/strapi/strapi/pull/442/commits/d9198dd225dcd4f37c8615031e48ce19208d6e36
Do you prefer to only lowercase the email?

Reply:
Yes, it's a mistake on our side. We should also remove the spaces, accents, etc by using the [_.deburr()](https://lodash.com/docs/4.17.4#deburr) method of Lodash. But, we have to keep uppercase characters.

Reply:
ok so I'll apply lowercase only in email field.

Reply:
@lucaperret Awesome! We will be able to merge after these changes.

Reply:
Ok but how to lowercase only the email field ?
Here https://github.com/strapi/strapi/pull/442/commits/d9198dd225dcd4f37c8615031e48ce19208d6e36 I don't know if it's an email or username...

Reply:
@lucaperret You have to add this line to the register method (https://github.com/strapi/strapi/blob/d9198dd225dcd4f37c8615031e48ce19208d6e36/packages/strapi-plugin-users-permissions/controllers/Auth.js#L32)